client/scss/components/forms/_error-message.scss

```diff
@@ -14,8 +14,12 @@
   // UI-Redesign: to be added via js and styled here
   // TODO Forms
   &::before {
-    font-family: $font-wagtail-icons;
+    content: '';
+    display: inline-block;
+    width: 1em;
+    height: 1em;
     vertical-align: -10%;
-    content: map.get($icons, 'warning');
+    mask-image: url('#{$images-root}icons/warning.svg');
+    background-color: currentColor;
```

Comment:
Same as `-after` icons – a lot of people have "error-message" directly referenced in their templates, and we don’t have a reusable component for this, so keeping the icon in CSS for now.

Reply:
Hmm I don’t understand what it’s about so sounds sensible to me.

Reply:
I assume it's about how error message elements for StreamField are constructed? If I remember correctly, they're dynamically created on the client-side via JS as opposed to server-rendered.